src/transformStateWithClones.js

```diff
@@ -65,7 +65,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const result = [];
+  const stateCopy = { ...state };
+
+  for (const selectedTransform of transforms) {
```

Comment:
```suggestion
  for (const { operation, properties } of transforms) {
```
Можна зробити деструктуризацію, і тоді не доведеться писати selectedTransform.properties ось таку звірюку, а можна буде просто properties / operation